flow/table.go

```diff
@@ -420,7 +427,18 @@ func (ft *Table) processFlowOP(op *Operation) {
 		fl.Metric.BAPackets += op.Flow.Metric.BAPackets
 
 		fl.Last = op.Flow.Last
-
+		if fl.Transport != nil {
```

Comment:
`if fl.Transport != nil && fl.Transport.Protocol == FlowProtocol_TCP && fl.TCPMetric != nil` 

Can you remove my `NOTE` line 422

Reply:
done